config/config.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/tidepool-org/platform/Godeps/_workspace/src/github.com/jinzhu/configor"
+)
+
+//FromJSON will load the requested JSON config into the given config interface
+func FromJSON(config interface{}, name string) error {
+
+	// we need to get the absolute path and also test it because of the
+	// difference of how this code runs in different environments
+	const configPath = "./_config/"
+	absPath, _ := filepath.Abs(configPath + name)
+	_, err := os.Open(absPath)
+
+	if err != nil {
+		const configLibPath = "./../_config/"
+		absPath, _ = filepath.Abs(configLibPath + name)
+	}
+
+	return configor.Load(&config, absPath)
+}
+
+//FromEnv will load the requested config value
+func FromEnv(name string) (string, error) {
```

Comment:
So, this only loads a string config from env? Do we have an JSON structs that are defined in the ENV?  (Probably not, but I know other repos have them like Jellyfish, for example.)